source/fuzz/fuzzer_pass_wrap_vector_synonym.cpp

```diff
@@ -33,8 +33,9 @@ void FuzzerPassWrapVectorSynonym::Apply() {
              opt::BasicBlock::iterator instruction_iterator,
              const protobufs::InstructionDescriptor& instruction_descriptor)
           -> void {
-        // Only run fuzzer pass on valid arithmetic operation instruction.
-        if (!valid_arithmetic_types.count(instruction_iterator->opcode()))
+        // Only run fuzzer pass on supported scalar operation type instruction.
+        if (!TransformationWrapVectorSynonym::OpcodeIsSupported(
+                instruction_iterator->opcode()))
           return;
```

Comment:
In the spirv-fuzz part of the code base we prefer to always use braces with if statements.

The C++ Google Style has an exceptional case in which curly braces can be omitted - see "For historical reasons" [here](https://google.github.io/styleguide/cppguide.html#Conditionals).

However, this case doesn't fit that exception. And anyway, we still prefer not to allow the exception.

Reply:
Done.